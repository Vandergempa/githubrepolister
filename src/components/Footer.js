import React from "react"

const Footer = () => {
  return (
    <footer className="pl-5 pr-5 d-flex" style={{ height: "150px" }}>
      <p className="m-auto pt-3 w-100 text-center border-top">Â© 2019 Made By Tamas Bogdan</p>
    </footer>
  )
}

export default Footer