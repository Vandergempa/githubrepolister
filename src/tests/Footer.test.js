import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/Footer';

test('Should render Footer component correctly', () => {
  let wrapper = shallow(
    <Footer />
  );
  expect(wrapper).toMatchSnapshot();
});