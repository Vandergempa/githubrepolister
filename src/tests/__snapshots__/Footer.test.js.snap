// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer component correctly 1`] = `
<footer
  className="pl-5 pr-5 d-flex"
  style={
    Object {
      "height": "150px",
    }
  }
>
  <p
    className="m-auto pt-3 w-100 text-center border-top"
  >
    Â© 2019 Made By Tamas Bogdan
  </p>
</footer>
`;
